src/cryptography/x509/name.py

```diff
@@ -420,7 +420,7 @@ def parse(self) -> Name:
             self._read_char(",")
             rdns.append(self._parse_rdn())
 
-        return Name(rdns)
+        return Name(reversed(rdns))
```

Comment:
Can you add a comment here referencing the text from the RFC, so we remember why we're reversing one?

Reply:
Ok. Done :-)